src/main/java/jenkins/plugins/nodejs/tools/pathresolvers/LatestInstallerPathResolver.java

```diff
@@ -114,10 +117,17 @@ public String resolvePathFor(String version, Platform platform, CPU cpu) {
             arch = "x64";
             break;
         case arm64:
+            if (nodeVersion.compareTo(new NodeJSVersion(8, 6, 0)) == 0 || nodeVersion.compareTo(new NodeJSVersion(4, 0, 0)) < 0) {
```

Comment:
Why 8.6.0 is excluded from download?

Reply:
This was already part of the initial code base
https://github.com/jenkinsci/nodejs-plugin/blob/master/src/main/java/jenkins/plugins/nodejs/tools/pathresolvers/LatestInstallerPathResolver.java#L118

Reply:
Yes but the baseline was wrong. For arm64
https://nodejs.org/dist/v8.6.0/node-v8.6.0-linux-arm64.tar.gz

Reply:
I removed the baseline.
Kept the case arm64 and armv7l as seperate cases (although the body is the same)